plugins/vpc-branch-eni/e2eTests/vlan_e2e_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should this file be called just "e2e_test". Just for consistency with AppMesh e2e tests.

Reply:
updated